lib/chef_zero/rspec.rb

```diff
@@ -1,5 +1,5 @@
 require "tempfile"
-require "chef_zero/server"
+require "chef_zero/ser1ver"
```

Comment:
```suggestion
require "chef_zero/server"
```

Reply:
Oops, typo somewhere when adding the line, sorry about that :(